mk/mod.mk

```diff
@@ -33,36 +33,36 @@ ifeq ($(STATIC),)
 
 $(MOD)$(MOD_SUFFIX): $($(MOD)_OBJS)
 	@echo "  LD [M]  $@"
-	$(HIDE)$(LD) $(LFLAGS) $(SH_LFLAGS) $(MOD_LFLAGS) \
-		$($(basename $@)_OBJS) \
-		$($(basename $@)_LFLAGS) -L$(LIBRE_SO) -lre -o $@
+	$(HIDE)$(LD) $(LFLAGS) $(SH_LFLAGS) $(MOD_LFLAGS) $($(MOD)_OBJS) \
+		$($(MOD)_LFLAGS) -L$(LIBRE_SO) -lre -o $@
```

Comment:
what is the reason for changing the `basename $@` parts here, is it related to this patch ?